services/telemetry/telemetry.go

```diff
@@ -346,7 +350,7 @@ func (s *Service) sendV2Request(ctx context.Context, req *reporter.ReportRequest
 		accessToken = ssoDetails.AccessToken.AccessToken
 	}
 
-	reqByte, err := json.Marshal(req)
+	reqByte, err := protojson.MarshalOptions{}.Marshal(req)
```

Comment:
if we are not setting any options can't we just use `protojson.Marshal`?  https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson#Marshal

Reply:
Good point. Yes in this package it is possible to use it like this. Changed.